### PR TITLE
Cleanup of device info

### PR DESCRIPTION
**Pull request by pmuetschard**
Removes the GLES info from device info as we don't need it and cleans up the code.

### CODE REVIEW

File: core/os/device/deviceinfo/cc/cpu.cpp [hunk 1]
```diff
@@ -18,55 +18,81 @@
...
   }
-  return "";
-}
 
-const char* cpuVendor() {
-  static union {
-    uint32_t reg[3];
-    char str[13];
-  };
+  str[12] = 0;
```
Comment (hevrard) [inline core/os/device/deviceinfo/cc/cpu.cpp:38]: Shouldn't this be `str[48] = 0;` ?

File: core/os/device/deviceinfo/cc/query.cpp [hunk 2]
```diff
@@ -72,36 +72,35 @@ void deviceInstanceID(device::Instance* instance) {
...
-  const char* backupName = "";
-  if (query::hasGLorGLES()) {
-    // Instance.Configuration.Drivers.OpenGLDriver
-    auto opengl_driver = new OpenGLDriver();
-    query::glDriver(opengl_driver);
-    drivers->set_allocated_opengl(opengl_driver);
-    backupVendor = opengl_driver->vendor().c_str();
-    backupName = opengl_driver->renderer().c_str();
-  }
+  const char* gpuVendor = "";  // TODO: we no longer have a vlue for this
```
Comment (hevrard) [inline core/os/device/deviceinfo/cc/query.cpp:102]: typo: vlue -> value

Is this something we should get via Vulkan now? (doesn't have to be in this PR)

File: core/os/device/deviceinfo/cc/cpu.cpp (hunk 1 above)
Comment (chrisforbes) [reply core/os/device/deviceinfo/cc/cpu.cpp:38]: No -- this is null termination for the old leaf 0 ("GenuineIntel" etc), which is exactly 12 characters long and does not include its own terminator

Comment (pmuetschard) [reply core/os/device/deviceinfo/cc/cpu.cpp]: Chris is correct, this is due to reusing the same buffer. If the vendor below uses exactly 12 bytes (without a terminating null), we wouldn't want it to also use the bytes from the device name.

I've added a comment.

File: core/os/device/deviceinfo/cc/query.cpp (hunk 2 above)
Comment (pmuetschard) [reply core/os/device/deviceinfo/cc/query.cpp]: I've pushed another commit with a tiny PCI ID database to lookup the vendor name.

**Pull request merged**